### ISSUE TITLE
The BNinception model

### ISSUE THREAD

**Issue by queenie88** [Cadene/pretrained-models.pytorch#96]
I trained with the bninception model. But I still face this problem when I trained on my datasets.
0 Epoch, 2900 Iter,Time :0.189,Data:0.012 Loss: 2.6701, Acc: 0.0469
0 Epoch, 3000 Iter,Time :0.189,Data:0.012 Loss: 2.6671, Acc: 0.0625
0 Epoch, 3100 Iter,Time :0.189,Data:0.012 Loss: 2.6090, Acc: 0.0469
0 Epoch, 3200 Iter,Time :0.189,Data:0.012 Loss: 2.5923, Acc: 0.0781
0 Epoch, 3300 Iter,Time :0.189,Data:0.012 Loss: 2.6018, Acc: 0.0781
0 Epoch, 3400 Iter,Time :0.188,Data:0.012 Loss: 2.5977, Acc: 0.0938
We can see the loss is not reduced and the acc is not to improve. If I change the lr, I also face the nan problem. I also find the example about the file of voc2007_extract.py. This model can be extracted the feature. But when I use this model to train with the public dataset, it is still facing the loss is not descend or nan problem.  I use other models, such as xception is not face these problems.

**Comment by Cadene** [comment]
Should be fixed: https://github.com/Cadene/pretrained-models.pytorch/commit/87e97515ec4480844e188d82e87e06376337a3af
